Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: Content items of a type without an icon get the default icon
+        Given a custom Content Type exists
```

Comment:
I'd remove "exists": `Given a custom Content Type".

Also, "custom" is imho meaningless in this context. If I've understood the feature correctly, it could go like this:

`Given a Content Type without a defined "ez-contenttype-icon-<identifier>" css class`